### ISSUE TITLE
ctrl-h / ctrl-u don't work after autocomplete

### ISSUE THREAD

**Issue by erg** [factor/factor#2524]
On macOS:

```
kevent[tab][enter]
ctrl-h or ctrl-u ! should show help / usage list
```

Maybe the autocomplete could use the vocab so it's in scope, or ctrl-h should be stronger and throw a restart or use the vocab.

Related to the autocomplete work with qualified names.

**Comment by mrjbq7** [comment]
doesn't work when the symbol isn't available in the manifest yet?

**Comment by erg** [comment]
Yes, this might have always been the same way. So maybe not a regression, but still annoying.

**Comment by mrjbq7** [comment]
Fixed by 8308cb629844e5fb7b97731f76ae6d6473113330.
